resources/htmlTemplate/ExceptionsTocHTMLTemplate.html

```diff
@@ -135,14 +135,17 @@ <h1>License Exceptions</h1>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-data"><strong>Data Files</strong></a>:  Machine readable files describing all of the licenses and license exceptions.</li>
-            <li style="margin:0px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/license-list-overview"><strong>Overview</strong></a>: General information about the SPDX License List, including principles for inclusion of a license and an explanation of the fields contained on the list.</li>
-            <li style="margin:5px 0 5px 10px;"><a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>Matching Guidelines</strong></a>:  Guidelines for what constitutes a license match to the SPDX License List. For licenses that include markup, the license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
-            <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Request New License</strong></a>: Instructions for proposing a license or exception be added to the SPDX License List.</li>
+            <li style="margin:5px 0 5px 10px;"><a href="index.html"><strong>SPDX License List</strong></a></li>
+            <li style="margin:5px 0 5px 10px;">The <a href="http://spdx.org/spdx-license-list/matching-guidelines"><strong>matching guidelines</strong></a> define what constitutes a license or exception match. The license text on the HTML pages here will display omitable text in blue and replaceable text in red (see Guideline #2 for more information).</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md"><strong>Explanation of fields</strong> used on the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md"><strong>License inclusion principles</strong> for adding new licenses or exceptions to the SPDX License List</li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md"><strong>Contribute</strong> to the project or request a new license</li>
+	      <li style="margin:0px 0 5px 10px;">Use <a href="http://spdx.org/ids"><strong>short identifiers in your source code</strong></a></li>
+	      <li style="margin:5px 0 5px 10px;"><a href="https://github.com/spdx/license-list-XML"><strong>Github repo</strong></a></li>
+            <li style="margin:5px 0 5px 10px;">Machine readable <a href="https://github.com/spdx/license-list-data"><strong>data files</strong></a> for the SPDX License List</li>
 
       </ul>
+      </ul>
```

Comment:
I think this additional `</ul>` is duplicative and shouldn't be added, unless I'm misreading?